app/Http/Controllers/Api/ContestsController.php

```diff
@@ -19,13 +25,37 @@ class ContestsController extends ApiController
     public function __construct()
     {
         $this->transformer = new ContestTransformer;
+        $this->contestAndWaitingTransformer = new ContestAndWaitingTransformer;
```

Comment:
So I think you'd just pass the contest data into the ContestTransformer and waiting room data into the WaitinRoomTransformer instead of creating a new transformer that repeats dealing with that data... @DFurnes mind chiming in?